lambda/value_rec_compiler.ml

```diff
@@ -349,6 +349,11 @@ let compute_static_size lam =
     | Patomic_compare_exchange _
     | Patomic_cas _
     | Patomic_fetch_add
+    | Patomic_add
+    | Patomic_sub
+    | Patomic_land
+    | Patomic_lor
+    | Patomic_lxor
```

Comment:
Note that you could have placed these primitives in the unit-returning section earlier if you wanted (It wouldn't change anything in practice since `Value_rec_check` wouldn't allow these cases).

Reply:
Sure, moved